apps/api/src/schema/websoc.ts

```diff
@@ -118,30 +136,73 @@ export const websocQuerySchema = z.object({
       }
       return parsedNums;
     }),
-  instructorName: z.string().optional(),
-  days: daysSchema.optional(),
-  building: z.string().optional(),
-  room: z.string().optional(),
+  instructorName: z.string().optional().openapi({
+    description: "The instructor's name of the WebSoc section",
+    example: "Anteater, P.",
+  }),
+  days: daysSchema.optional().openapi({
+    description: "Selects WebSoc sections held on any of the given days",
+    example: "M,Tu,W,Th",
+  }),
+  building: z.string().optional().openapi({
+    description: "The building of the WebSoc section",
+    example: "ALP",
+  }),
+  room: z.string().optional().openapi({
+    description: "The room of the WebSoc section",
+    example: "1100",
+  }),
   division: z
     .enum(courseLevels)
     .or(z.literal("ANY"))
     .optional()
+    .openapi({
+      description: "The division level of the WebSoc section",
+      example: "LowerDiv",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   sectionType: z
     .union([z.enum(anyArray), z.enum(websocSectionTypes)])
     .optional()
+    .openapi({
+      description: "The section type of the WebSoc section, see [Registrar](https://reg.uci.edu/)",
+      example: "Lec",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   fullCourses: z
     .enum(fullCoursesOptions)
     .optional()
+    .openapi({
+      description: "Filter WebSoc sections by filled status",
+      example: "SkipFull",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
-  cancelledCourses: z.enum(cancelledCoursesOptions).optional(),
-  units: z.optional(z.literal("VAR").or(z.string())),
-  startTime: timeSchema.optional(),
-  endTime: timeSchema.optional(),
+  cancelledCourses: z.enum(cancelledCoursesOptions).optional().openapi({
+    description:
+      "Filter courses based on their cancellation status, including only cancelled courses",
+    example: "Exclude",
+  }),
+  units: z.optional(z.literal("VAR").or(z.string())).openapi({
+    description:
+      "Filter WebSoc sections by unit count. The value 'VAR' is used for courses with variable unit values, where the unit count can vary depending on the course section",
+    example: "4",
+  }),
+  startTime: timeSchema.optional().openapi({
+    description: "Filters WebSoc sections for courses that start at or after this time",
```

Comment:
This can be more concise. In particular saying both "sections" and "courses" produces added confusion.